tox.ini

```diff
@@ -9,14 +9,14 @@ deps=
 setenv =
     PYTHONPATH = {toxinidir}
 commands =
-    pytest {posargs} armi
+    pytest --ignore=armi/utils/tests/test_gridGui.py {posargs} armi
```

Comment:
If we use the environment variable-based `skipif` thingy, we can disable the GUI tests by deglecting to set the `ARMI_GUI_TESTS` variable

Reply:
A little worried whether the `import wx` statement would crash the test in Travis, I'll try it with and without the ignore statements, and see if we can do without them.

Reply:
It appears that we can't do without the `--ignore` flags:
- With `--ignore`: https://coveralls.io/builds/36415490
- Without `--ignore`: https://coveralls.io/builds/36415610

Reply:
yeah, no kidding. okay, we can stomp on this later